helm/adelphi/templates/nfs-server.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: nfs-server
+  namespace: {{ .Values.namespace }}
+spec:
+  replicas: 1
+  selector:
+    role: nfs-server
+  template:
+    metadata:
+      labels:
+        role: nfs-server
+    spec:
+      containers:
+      - name: nfs-server
```

Comment:
this server is here to overcome the ReadWriteMany limitation in GKE, while keeping the workflow self-contained and not depending on extra operational work